static/styles/tooltips.css

```diff
@@ -64,11 +64,15 @@
 
     .hotkey-hint {
         box-sizing: border-box;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
```

Comment:
Do these CSS changes belong in the previous commit that defines this class?

Reply:
While resolving the design queries, these attributes are no longer valid. But as you mention, the CSS had been a little mixed up due to error in rebasing, while trying to solve a merge conflict.